build/Targets/SettingsSdk.props

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <RoslynSdkProject>true</RoslynSdkProject>
     <EnableDefaultItems>false</EnableDefaultItems>
+    <OptionStrict>Off</OptionStrict>
+    <NoWarn>$(NoWarn);41999;42016</NoWarn>
```

Comment:
Believe we need to do this setting at the bottom of `Settings.props`.  It has to come after the import of the SDK props file or they will simply overwrite the settings.   Here is as section where we already do this for a few other properties.

https://github.com/dotnet/roslyn/blob/master/build/Targets/Settings.props#L242